pkgs/development/libraries/liblangtag/default.nix

```diff
@@ -39,12 +39,12 @@ stdenv.mkDerivation rec {
     unzip ];
   nativeBuildInputs = [ autoreconfHook gtk-doc gettext pkgconfig ];
 
-  meta = {
+  meta = with stdenv.lib; {
```

Comment:
please avoid making these kind of style changes, because they're typically personal.